config.go

```diff
@@ -404,13 +406,26 @@ func ColumnBufferSize(size int) RowGroupOption {
 // The order of sorting columns passed as argument defines the ordering
 // hierarchy; when elements are equal in the first column, the second column is
 // used to order rows, etc...
-func SortingColumns(sortingColumns ...SortingColumn) RowGroupOption {
+func SortingColumns(columns ...SortingColumn) interface {
```

Comment:
Why wouldn't this just return []SortingColumn?

Reply:
Ohhh... the RowGroupOption is part of the return interface, that's not super intuitive. I'd maybe pull that out, `type RowGroupWriterOption` (?)

Reply:
The return type needs to implement the option interfaces to be used as input argument to `parquet.NewWriter` and such.